INSTALL

```diff
@@ -486,6 +486,13 @@ MFEM_USE_CEED = YES/NO
    library for performant high-order operator evaluation developed by the Center
    for Efficient Exascale Discretizations in the Exascale Computing Project.
 
+MFEM_USE_MKL_CPARDISO = YES/NO
```

Comment:
Need to add also a section under "External Libraries" with URL, version requirements, etc.